### PR TITLE
Transform route titles into real h1's, for #1651

### LINKED ISSUE #1651: Document has no headline level one
## Description
Parts of document needs to have headings to provide a better navigation through the page with assistive technologies. Sometimes needs to be then visually hidden though.

### PR DESCRIPTION
**Pull request by marcus-herrmann**
## Description
Transformed spans the held the page title to actual h1 elements. In the case of the file list a h1 is existing for accessibility reasons, but can only be perceived via a screen reader (is "visually-hidden").

## Related Issue

- Fixes #1651

## Motivation and Context
Screen reader users navigate by large extent by a documents or web apps headline structure. At least one level 1 headline should exist to make the pages (routes) purpose, content or function clear.

## How Has This Been Tested?
*work in progress*

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Tests

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Account</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPermissionsUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>



















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>iPhone</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingAutocompletion</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>














</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Resharing</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/7145/

<p>





</p></details>

**Comment by PVince81** [comment]
Rebased.

@LukasHirt please review

File: apps/files/src/components/FilesAppBar.vue [hunk 1]
```diff
@@ -8,7 +8,10 @@
         </div>
         <span class="uk-flex uk-flex-middle" v-if="!showBreadcrumb">
           <oc-icon v-if="pageIcon" :name="pageIcon" class="uk-margin-small-right" />
-          <span class="uk-text-lead">{{pageTitle}}</span>
+          <h1 class="oc-page-title">{{pageTitle}}</h1>
```
Comment (LukasHirt) [inline apps/files/src/components/FilesAppBar.vue]: Would be nice to use `v-text`

File: apps/files/src/components/FilesAppBar.vue [hunk 2]
```diff
@@ -8,7 +8,10 @@
...
+        </span>
+        <span v-else-if="showBreadcrumb">
+          <h1 class="oc-visually-hidden">{{pageTitle}}</h1>
```
Comment (LukasHirt) [inline apps/files/src/components/FilesAppBar.vue]: Would be nice to use `v-text`

**Comment by LukasHirt** [comment]
@PVince81 I adjusted the PR to use `v-text` and to use ID as the selector to get this PR again going. Pls review 🙂

**Pull request merged**